www/src/utils/createOgUrl.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import {urlRoot} from 'site-constants';
+
+export default markdownPath => {
+  const path = markdownPath ? markdownPath.replace(/\.md$/, '.html') : '';
```

Comment:
Can you add a comment explaining what you're trying to do here?

Reply:
Add a comment and remove an unnecessary null check.